testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -7045,6 +7049,33 @@ void checkRegexpExtract(SqlOperatorFixture f0, FunctionAlias functionAlias) {
     f0.forEachLibrary(list(SqlLibrary.MYSQL, SqlLibrary.SPARK), consumer);
   }
 
+  /** Test case for
+   * <a href="https://issues.apache.org/jira/browse/CALCITE-6321">[CALCITE-6321]
+   * Add LOG function (enabled in PostgreSQL library)</a>. */
+  @Test void testPostgresLogFunc() {
+    final SqlOperatorFixture f0 = fixture()
+        .setFor(SqlLibraryOperators.LOG_POSTGRES, VmName.EXPAND);
+    f0.checkFails("^log(100, 10)^",
+        "No match found for function signature LOG\\(<NUMERIC>, <NUMERIC>\\)", false);
+    final SqlOperatorFixture f = f0.withLibrary(SqlLibrary.POSTGRESQL);
+    f.checkScalar("log(10, 10)", 1.0,
+        "DOUBLE NOT NULL");
+    f.checkScalar("log(64, 8)", 2.0,
```

Comment:
LOG(base, value), the result of log(64, 8) is not 2.0

Reply:
ohhh, seems to be a bug in sqlfunctions, I open a new jira case, bigquery is log(64, 8) == 2?
I overlooked the problem, and I'm sorry

Reply:
fix